core/src/main/java/org/mule/api/registry/ServiceRegistry.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.api.registry;
+
+import java.util.Collection;
+import java.util.ServiceLoader;
+
+/**
+ * API component to be used as standard SPI discovery mechanism.
```

Comment:
This is not a "SPI discovery mechanism", that is the implementation. This is a "discovery mechanism"